### ISSUE TITLE
redirect 0.118 or later

### ISSUE THREAD

**Issue by quozl** [sugarlabs/aslo#21]
Clients running a user agent of Sugar 0.118 or later to be redirected to v4 when they hit https://activities.sugarlabs.org/

(This is to handle those who haven't got a new release of Browse).

@srevinsaju?

**Comment by srevinsaju** [comment]
Yes, agreed. We need to add that. I will attempt to create a pull request, I am not well educated with PHP, so I didnt touch this earlier :stuck_out_tongue:
